decidim-conferences/app/cells/decidim/conferences/conference_speaker_cell.rb

```diff
@@ -66,13 +66,13 @@ def short_bio
       def twitter_handle
         return unless model.twitter_handle.presence
 
-        link_to t(".go_to_twitter"), "https://twitter.com/#{model.twitter_handle}", target: "_blank", rel: "noopener"
+        link_to "@#{model.twitter_handle}", "https://twitter.com/#{model.twitter_handle}", target: "_blank", rel: "noopener"
       end
 
       def personal_url
         return unless model.personal_url.presence || (model.user.presence && model.user.personal_url.presence)
 
-        link_to t(".personal_website"), model.personal_url || model.user.personal_url, target: "_blank", class: "card-link", rel: "noopener"
```

Comment:
If we delete the text on the link_to call the method will fail. I'm going to recover the argument